### DISCUSSION TITLE
Unreal Engine 5 Integration

### DISCUSSION THREAD

**Discussion by intelliot** [XRPLBounties/Proposals#14] [Open Bounties]
[Sorry, this bounty is now closed. For other opportunities, see [Funding Programs](https://xrpl.org/developer-funding.html) and join the [XRPL Developers Discord server](https://xrpldevs.org/).]

**Comment by ZachDolph** [comment]
Hello XRPL Team,

I’d like to work on the Unreal integration plugin, and can have it completed this week with a demo as well. Before starting, I wanted to make sure that someone else hasn’t already began working on it or expressed interest. Thank you!

Zach

**Reply by JST5000** [reply]
This bounty is still available to be picked up - I personally haven't seen any comments expressing interest in it but can't say for certain no one is working on it.

**Reply by intelliot** [reply]
By the way, it's not necessarily a problem if someone else is working on it. We have the ability to pay the full bounty to multiple bounty hunters / developers (if they develop high-quality, independent implementations).

**Comment by dangell7** [comment]
@ZachDolph I haven't seen anyone working on the UE plugin. I'm in the process of getting the Unity plugin accepted in the asset store.

I did find this [repository](https://github.com/ripple/ripple-libpp) for c++ that wraps the rippled repo functionality so thats probably a good sdk to use.

XRPL bounty managers can comment more.

**Reply by ZachDolph** [reply]
Hey @dangell7 I appreciate your response! Thank you for the share on the ripple cpp library, coincidentally I was just writing a small wrapper around curl and a json parser, so this is perfect.

Good luck on your Unity plugin! I’ll let you know when this is completed so you can check it out.

**Comment by 3scava1i3r** [comment]
Is this still up for grabs?

**Reply by intelliot** [reply]
Sorry, this bounty is now closed. Thank you for your interest!

**Comment by 3scava1i3r** [comment]
Understood is there any other XRPL bounties i could work on?

**Reply by intelliot** [reply]
While there are no XRPL Bounties currently open for submissions, you may be interested in other [XRPL funding programs](https://xrpl.org/developer-funding.html). Additionally, you're always more than welcome to work on open source projects, and in that regard, we're always happy to answer any specific questions you have.

**Reply by wojake** [reply]
Is the bounty program no longer in service?

**Comment by 3scava1i3r** [comment]
> While there are no XRPL Bounties currently open for submissions, you may be interested in other [XRPL funding programs](https://xrpl.org/developer-funding.html). Additionally, you're always more than welcome to work on open source projects, and in that regard, we're always happy to answer any specific questions you have.

Understood, I have some valuable insights and suggestions to enhance this proposal. Shall we continue our discussion here, or would you prefer to communicate via Discord for a more detailed conversation?

**Reply by intelliot** [reply]
Can you create a new Discussion in [Ideas](https://github.com/XRPLBounties/Proposals/discussions/categories/ideas)?
